services/src/main/java/org/keycloak/services/resources/admin/ClientResource.java

```diff
@@ -565,6 +566,9 @@ public void registerNode(Map<String, String> formParams) {
         if (node == null) {
             throw new BadRequestException("Node not found in params");
         }
+        
+        ReservedCharValidator.validate(node);
```

Comment:
I think node is the hostname of a node, so an IP address or domain, with/without ports. So this would probably break things as ':' is marked as a reserved character.

@mposolda ??

Reply:
Yes, it is hostname of a node, so an IP address or domain. However it is without ports. So this validation should be ok.